docs/README.md

```diff
@@ -1920,60 +1914,277 @@ mutation {
+
+// Send a command (mutation)
+const commandResult = await client.mutate({
+  variables: {
+    cartID: "demo",
+    sku: "ABC_02",
+  },
+  mutation: gql`
+    mutation AddOneItemToCart($cartID: ID!, $sku: string!) {
+      ChangeCart(input: { cartId: $cartID, sku: $sku, quantity: 1 })
+    }
+  `,
+})
+
+// Subscribe to changes in the CartReadModel
+const subscriptionOperation = client.subscribe({
+  variables: {
+    cartID: "demo",
+  },
+  query: gql`
+    subscription SubscribeToCart($cartID: ID!) {
+      CartReadModel(id: $cartID) {
+        id
+        cartItems
+      }
+    }
+  `,
+})
+
+subscriptionOperation.subscribe({
+  next: (cartReadModel) => {
+    // This function is called everytime the CartReadModel with ID="demo" is changed
+    // Parameter "cartReadModel" contains the latest version of the cart
+  }
+})
+```
+
+#### Authorizing operations
+
+When you have a command or read model whose access is authorized to users with a specific set of roles (see ["Authentication and Authorization"](#authentication-and-authorization)), you need to use an authorization token when 
+sending queries, mutations or subscriptions to that command or read model. See the ["Authentication API"](#authentication-api) and, more especifically, the ["Sign in"](#sign-in) section to know how to get a token.
+
+Once you have a token, the way to send it varies depending on the protocol you are using to send GraphQL operations:
+- For **HTTP**, you need to send the HTTP header `Authorization` with the token, making sure you prefix it with `Bearer ` (the kind of token Booster uses). For example:
+```http request
+Authorization: Bearer <your token>
+```
+- For **WebSocket**, you need to adhere to the ["GraphQL over WebSocket" protocol](#the-graphql-over-websocket-protocol) to send authorization data. The way to do that is by sending the token in the payload of the first message you send when initializing the connection (see ["Subscribing to read models](#subscribing-to-read-models)). For example:
+```json
+ { "type": "connection_init", "payload": { "Authorization": "<your token>" } }
 ```
 
-Once the client is setup, a subscription to read model GraphQL request through a GraphQL client could be:
+You normally won't be sending tokens in such a low-level way. GraphQL clients has easier ways to send these tokens. See ["Sending tokens with Apollo client"](#sending-tokens-with-apollo-clients) 
+
+##### Sending tokens with Apollo clients
+
+We recommend going to the specific documentation of the specific Apollo client you are using to know how to send tokens. However, the basics remains the same. Here is an example of how you would configure the Javascript/Typescript Apollo client to send the authorization token. The example is exactly the same as the one shown in section ["Using Apollo clients"](#using-apollo-client) but with the changes needed to send the token. Notice that the only things that change are the `HttpLink` and the `WebSocketLink`:
 ```typescript
```

Comment:
The same with this code. If it is available on apollo website I would remove, unless we are here explaining something very tricky and uncovered in the official documentation.

Reply:
Check https://github.com/boostercloud/booster/pull/375#discussion_r473982082